sam's datapackage blog

swilairat opened the pull request:
# Overview

Hi Lilly, 

Here's my data package blog! 


-Sam

---

Please preserve this line to notify @lwinfree (lead of this repository)

lwinfree reviewed (changes requested):
Hi Sam! This looks good, but there are a few things to change. 
Your image needs to be added with HTML instead of Markdown due to the styling of the site. Here's an example of how that works:
```
<img src="your-image.png" width=100% alt="My Image Name">
<p style="text-align: center; font-style: italic">This is a description of my image.</p>
```
And there are 2 typos: 
- line 20: "Knolwedge"
- line 30: you need a space between the data package creator link & the word "yourself"

Please make changes and add them with a new commit to this same branch. They will automatically be added to this pull request when you commit them to your branch. Ping me when it is fixed & I will take a look :-) 

Let me know if you need help or have questions!

swilairat commented:
> Hi Sam! This looks good, but there are a few things to change.
> Your image needs to be added with HTML instead of Markdown due to the styling of the site. Here's an example of how that works:
> 
> ```
> <img src="your-image.png" width=100% alt="My Image Name">
> <p style="text-align: center; font-style: italic">This is a description of my image.</p>
> ```
> 
> And there are 2 typos:
> 
> * line 20: "Knolwedge"
> * line 30: you need a space between the data package creator link & the word "yourself"
> 
> Please make changes and add them with a new commit to this same branch. They will automatically be added to this pull request when you commit them to your branch. Ping me when it is fixed & I will take a look :-)
> 
> Let me know if you need help or have questions!

Hi Lilly, 

I updated the photo format and fixed the typos.

Thanks, 

Sam

lwinfree reviewed (approved):
Looks good now! Thanks!
